qutebrowser/browser/qtnetworkdownloads.py

```diff
@@ -203,6 +203,17 @@ def _ask_confirm_question(self, title, msg):
                               no_action=no_action, cancel_action=no_action,
                               abort_on=[self.cancelled, self.error])
 
+    def _ask_create_parent_question(self, title, msg,
```

Comment:
Like @Kingdread mentioned, this should have a corresponding abstract method in `qutebrowser/browser/downloads.py`.